LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 ahasl
```

Comment:
Apparently the copyright owner needs to be a legal entity (i.e. full name or actual corporation required).